src/components/shared/manual-form.tsx

```diff
@@ -11,22 +12,27 @@ type EditableFormProps = {
 
 export const ManualForm = ({ field, title }: EditableFormProps) => {
   const { user, refetchUser } = useContext(AuthContext);
+  const { showSnackbar } = useContext(SnackbarContext);
```

Comment:
This works, but you could export a `useSnackbar()` or `useToast()` hook from `snackbar.context.tsx`, some thing like:
```
export function useToast() {
  return   useContext(SnackbarContext)
}
```
So anywhere else in the app, you can call that directly instead of ` useContext(SnackbarContext)`, making it clearer